ehr/resources/web/ehr/window/OpenSurgeryCasesWindow.js

```diff
@@ -336,7 +356,8 @@ EHR.DataEntryUtils.registerDataEntryFormButton('OPENSURGERYCASES', {
         }
 
         Ext4.create('EHR.window.OpenSurgeryCasesWindow', {
-            sourceStore: clientStore
+            sourceStore: clientStore,
+            caseUpdateStores: panel.storeCollection.clientStores.items,
```

Comment:
This is picky, but wouldnt it be a little more normal to make a pointer to the StoreCollection, rather than 'items', which is more like an internal field of that class? Either accomplishes your goal, obviously.